### ISSUE TITLE
demo/k8s: setup_k8s_cluster.sh-etcd hosts not firing up on other nodes

### ISSUE THREAD

**Issue by mpduarte** [contiv/demo#35]
it appears the /etc/etcd/etcd.conf file isn't being properly configured for start up.

Made the following modifications:
ETCD_INITIAL_CLUSTER=svlngen4-fab2-container1=http://netmaster:2380
to:
ETCD_LISTEN_CLIENT_URLS="http://0.0.0.0:4001"
ETCD_INITIAL_CLUSTER=svlngen4-fab2-container1=http://svlngen4-fab2-container1.cisco.com:2380

**Comment by mpduarte** [comment]
Issue is with centos/rhel 7.2.2 release
https://rhn.redhat.com/errata/RHBA-2016-0231.html
the service file for etcd has been updated, which now requires a listening port to be specified. 

I've manually fixed the issue, but will need to specify this line for future deployments. In addition I suggest for future ansible scripts, to call for specific versions of packages, which should mitigate this issue somewhat in the future.
